benchmark/FragmentDB_bench.jl

```diff
@@ -0,0 +1,32 @@
+function prepare_mol(fdb, steps)
+    sys = load_pdb(ball_data_path("../benchmark/data/aspirin_pug_bonds.json"))
+    #fdb = FragmentDB()
+
+    if steps > 0
+        normalize_names!(sys, fdb)
+    end
+
+    if steps > 1
+        reconstruct_fragments!(sys, fdb)
+    end
+
+    if steps > 2
+        build_bonds!(sys, fdb)
+    end
+    sys
+
+end
+
+
+fdb_suite = SUITE["FragmentDB"]
+
+fdb_suite["Creation"] = @benchmarkable FragmentDB()
+
+
+fdb = FragmentDB()
+
+fdb_suite["Normalize_Name"] = @benchmarkable normalize_names!(sys, $fdb) setup=(sys = prepare_mol($fdb,1))
+
+fdb_suite["Reconstruct_Fragments"] = @benchmarkable reconstruct_fragments!(sys, $fdb) setup=(sys = prepare_mol($fdb, 2))
+
+fdb_suite["Building_Bonds"] = @benchmarkable normalize_names!(sys, $fdb) (setup=(sys = prepare_mol($fdb, 3)))
```

Comment:
```suggestion
fdb_suite["Normalize_Name"] = @benchmarkable normalize_names!(sys, $fdb) setup=(sys = prepare_mol($fdb,0))

fdb_suite["Reconstruct_Fragments"] = @benchmarkable reconstruct_fragments!(sys, $fdb) setup=(sys = prepare_mol($fdb, 1))

fdb_suite["Building_Bonds"] = @benchmarkable normalize_names!(sys, $fdb) (setup=(sys = prepare_mol($fdb, 2)))
```
I think these args should be decreased by one. Otherwise, the benchmarked function is already applied during setup. Also, the used system does not seem to miss fragments or bonds, so the FragmentDB does not really reconstruct anything here. Other data (with incomplete bonds/fragments) might be more meaningful for benchmarking. I haven't checked whether `normalize_names` has anything to do here.

Reply:
Makes sense. I will search for another example.